pkg/controller/job/metrics/metrics.go

```diff
@@ -71,6 +71,20 @@ var (
 		[]string{"completion_mode", "result", "reason"},
 	)
 
+	// JobByExternalControllerTotal tracks the number of Jobs that were created
+	// as managed by an external controller.
+	// The value of the label controllerName corresponds to the value of the
+	// managed-by label.
+	JobByExternalControllerTotal = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Subsystem:      JobControllerSubsystem,
+			Name:           "jobs_by_external_controller_total",
+			Help:           "The number of Jobs managed by an external controller",
+			StabilityLevel: metrics.ALPHA,
```

Comment:
```suggestion
			StabilityLevel: metrics.BETA,
```

Why do we introduce this metric as an alpha level? Isn't the `JobManagedByLabel` feature at the beta level?

Reply:
I think the metrics have a separate graduation lifecycle than features, governed by the rules in this KEP: https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/1209-metrics-stability/kubernetes-control-plane-metrics-stability.md. 

The KEP actually says is is not clear if the "Beta" stability level is needed, see here: https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/1209-metrics-stability/kubernetes-control-plane-metrics-stability.md#beta-stability-level.

Whereas the "stable" level means we cannot add new labels: "no labels can be added or removed from this metric" quota, which may be too early for the new metric.

Reply:
That makes sense. As I investigated the history, after the BETA stage was added here https://github.com/kubernetes/kubernetes/pull/112676, the necessity of BETA seems to have been denied.